build.gradle

```diff
@@ -0,0 +1,331 @@
+/*
+ * Gradle file for the KBase workspace_deluxe.
+ */
+
+plugins {
+	id "java"
+	id "war"
+	id "jacoco"
+}
+
+group = "workspace service"
+version = "0.14.1"
+
+ext {
+	clientJarFile = "WorkspaceClient.jar"
+	serviceJarFile = "WorkspaceService.jar"
+	warFile = "WorkspaceService.war"
+	warXml = "war/web.xml"
+	// directories
+	src = "src"
+	dist = "dist"
+	jarDir = "../jars/lib/jars"
+	downloadDir = "$buildDir/download"
+	tempunpack = "$buildDir/tmp/unpackedjars"
+	// other config
+	kbaseCommonJar = "kbase-common-0.1.1.jar"
+	testConfig = "test.cfg"
+}
+
+repositories {
+	mavenCentral()
+}
+
+configurations {
+	implementation
+	testImplementation
+	runtimeDeps {
+		extendsFrom implementation
+	}
+	allDeps {
+		extendsFrom implementation, testImplementation
+	}
+}
+
+dependencies {
+	implementation files("$jarDir/kbase/common/$kbaseCommonJar")
```

Comment:
These are copied directly from the ant file. The next step will be removing some of the deps that either aren't used or are already supplied by an existing dependency.